canonical-web-and-design/ubuntu.com
# Account users/new user modal

## Done

- add NewUserForm within modal
  - add validation, formSubmissionError handling, success message
- remove "date added" column
- add name form field
- add @testing-library/user-event





## QA

1. Go to https://ubuntu-com-10348.demos.haus/advantage/users?test_backend=true and login
2. Add a new user
3. Verify that form validation requires all fields to have a value
4. Verify the success message has been displayed on successful submission

## Issue / Card

https://github.com/canonical-web-and-design/commercial-squad/issues/197

## Screenshots
### initial state
![image](https://user-images.githubusercontent.com/7452681/133050088-3f522602-7ac5-4f96-a2ae-3173b260cfc4.png)

### Success
![image](https://user-images.githubusercontent.com/7452681/133059683-98ebf6b5-06dc-43b2-bab5-c3dd19e53702.png)

### Error
![image](https://user-images.githubusercontent.com/7452681/133050713-510c42dd-799f-4640-9bc5-6ca3044f5adc.png)
